api/src/main/java/com/alibaba/nacos/api/naming/pojo/healthcheck/HealthCheckerFactory.java

```diff
@@ -76,11 +78,10 @@ public static AbstractHealthChecker deserialize(String jsonString) {
         try {
             return MAPPER.readValue(jsonString, AbstractHealthChecker.class);
         } catch (IOException e) {
-            // TODO replace with NacosDeserializeException.
-            throw new RuntimeException("Deserialize health checker from json failed", e);
+            throw new NacosDeserializationException("Nacos deserialize health checker from json failed", e);
```

Comment:
use `NacosDeserializationException(Class<?> targetClass, Throwable throwable)` please.

Reply:
Thanks for guidance